assets/cases/ui/21.video-player/video-player-ctrl.ts

```diff
@@ -81,6 +81,17 @@ export class VideoPlayerCtrl extends Component {
         }
     }
 
+    onEventType (target: VideoPlayerCtrl, type: string) {
+        switch (type) {
+            case VideoPlayer.EventType.READY_TO_PLAY: {
```

Comment:
From doc: https://docs.cocos.com/creator/manual/zh/ui-system/components/editor/videoplayer.html#%E5%85%B3%E4%BA%8E%E8%87%AA%E5%8A%A8%E6%92%AD%E6%94%BE%E7%9A%84%E9%97%AE%E9%A2%98 
Some platform like wechat-platform cannot auto-play after set video source. So we use READY_TO_PLAY Event to listen and call play to make sure video will play after emit READY_TO_PLAY

Reply:
This logic is used on all platforms. Will the platforms that can auto-play have problem with the logic?

Reply:
No, in other platforms video can auto-play and videoPlayer.isPlaying will set to `true` , so they won't run this logic.